movement/io/save_poses.py

```diff
@@ -361,6 +367,139 @@ def to_sleap_analysis_file(ds: xr.Dataset, file_path: str | Path) -> None:
     logger.info(f"Saved poses dataset to {file.path}.")
 
 
+def to_nwb_file(
+    ds: xr.Dataset, config: NWBFileSaveConfig | None = None
+) -> list[pynwb.file.NWBFile]:
```

Comment:
I tried using this function and I found it a bit counter-intuitive that this returns a list even if there is only 1 subject/NWBFile. I suggest changing it to return just the NWBFile for single-individual datasets, and List[NWBFile] only for multi-individual ones.

Reply:
I am probably to blame for this, but I have fresh perspective now 😄